src/runtime/runtime.js

```diff
@@ -177,6 +177,9 @@ export class Runtime {
     /** @private {?PageConfigResolver} */
     this.pageConfigResolver_ = null;
 
+    /** @private @const {!ClientEventManager} */
+    this.eventManager_ = new ClientEventManager(this.configured_(false));
```

Comment:
did you mean true?

Reply:
If I use true it breaks a bunch of tests.  I think it isn't ready to be called with true at this point.

Reply:
And just to note, calling it will false returns the same promise it would have either way - it just doesn't kick of resolution yet.

Reply:
makes sense. I don't think we will remember why we did it this way even a few months down, can you add comments?